packages/firestore/src/local/indexeddb_query_cache.ts

```diff
@@ -385,6 +385,27 @@ function globalTargetStore(
   );
 }
 
+function retrieveMetadata(
+  txn: SimpleDbTransaction
+): PersistencePromise<DbTargetGlobal> {
+  const globalStore = SimpleDb.getStore<DbTargetGlobalKey, DbTargetGlobal>(
+    txn,
+    DbTargetGlobal.store
+  );
+  return globalStore.get(DbTargetGlobal.key).next(metadata => {
+    assert(metadata !== null, 'Missing metadata row.');
+    return metadata!;
+  });
+}
+
+export function getHighestListenSequenceNumber(
```

Comment:
These exported global functions don't port well and "pollute" the global namespace. Where possible, can you attach them to a class, similar to what I assume will be the case in Android/iOS?